vscode-lean4/src/utils/fsHelper.ts

```diff
@@ -0,0 +1,39 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
+import { readFile, promises, PathLike } from 'fs';
+
+
+export async function fileExists(pathFile: PathLike): Promise<boolean> {
+    /*
+    Helper used to replace fs.existsSync (using existsSync to check for the existence
+    of a file before calling fs.open(), fs.readFile() or fs.writeFile() is not recommended.
+    Doing so introduces a race condition, since other processes may change the file's state between the two calls.
+    Instead, user code should open/read/write the file directly and handle the error raised if the file does not exist.)
+    param: pathFile - A string representing a PathLike
+
+    returns Promise<boolean> a boolean that represents if a file exist
+    */
+    return await promises.access(pathFile).then(() => true, () => false);
+}
+
+export async function fileRead(url: PathLike): Promise<string>{
```

Comment:
This is just `(await fsPromises.readFile(url, {encoding: 'utf-8'})).trim()`.  I don't think we need a helper function for this.  (The name is also slightly misleading, because I wouldn't expect that a generic `fileRead` function trims the file contents).

Reply:
Ok :) Resolved!